### PR TITLE
operator datasance-pot-operator (3.4.7)

### PR DESCRIPTION
**Pull request by emirhandurmus**

### CODE REVIEW

File: operators/datasance-pot-operator/3.4.7/manifests/datasance-pot-operator_deployment.yaml
Comment (acornett21) [inline operators/datasance-pot-operator/3.4.7/manifests/datasance-pot-operator_deployment.yaml:1]: Hi @emirhandurmus This deployment file is not needed, and is causing your issues. The deployment information is already in the CSV. Hope this helps.

Comment (emirhandurmus) [reply operators/datasance-pot-operator/3.4.7/manifests/datasance-pot-operator_deployment.yaml:1]: Hi  @acornett21 thanks for the response. I've delete deployment.yaml and restart the flow. This time it failed on preflight-trigger task which is weird that it wasnt happen before I deleted deployment.yaml and troubleshooting-guide refer it as an "uncommon". now trying to restart pipeline to see what happens

Comment (emirhandurmus) [reply operators/datasance-pot-operator/3.4.7/manifests/datasance-pot-operator_deployment.yaml:1]: Hi @acornett21 after I restart flow it passed and merge completed. Thanks a lot for the help

**Pull request merged**